gateway/manager/handler.go

```diff
@@ -68,7 +68,7 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	s.setContexts(r, workspace, token)
+	r = s.setContexts(r, workspace, token)
```

Comment:
Standard lint config doesn't catch ineffective assigning, maybe we should go with custom linter configuration?